22 спринт

David-Arcanone opened the pull request:
задание 22 спринта
- заготовки на 23 спринт (пока закомментировал, чтоб не мешали, находятся в пакедже com.practicum.playlistmaker.playlistOverview)

David-Arcanone committed:
спринт 22
промежуточная версия

David-Arcanone committed:
спринт 22
промежуточная версия

David-Arcanone committed:
спринт 22
изменил формат сохранения Плейлиста, теперь сохраняю список Id вместо списка треков.

David-Arcanone committed:
спринт 22
изменил формат сохранения Плейлиста, теперь сохраняю список Id вместо списка треков.

David-Arcanone committed:
спринт 22
подправил баг сbottomSheet на фрагменте плеера

kosstyango commented:
⚠️ Критические замечания:
При поиске слова "Проверка" приложение падает (см.скрин ошибок)
![Спринт_22_Григорьев3](https://github.com/user-attachments/assets/eb81bde5-4c7c-4a4c-b552-99d61c17a527)

🍏 Рекомендации, не влияющие на результат проверки работы:
При выходе из экрана Новый плейлист диалог есть, заголовок есть, сообщения нет. Кнопки "ДА" и "НЕТ" плохо видны в темной теме.
При успешном добавлении трека окно добавления трека в плейлист НЕ исчезает.

David-Arcanone committed:
спринт 22
устранил замечания согласно ревью:
-диалог закрытия при создании плейлиста (включая внешность кнопок)
-закрытие BottomSheet после успешного добавления трека в плейлист
-слово "Проверка" можно вбивать в поиск и программа не будет вылетать
